website/docs/reference/snapshot-configs.md

```diff
@@ -150,11 +150,27 @@ Configurations can be applied to snapshots using the [YAML syntax](/docs/build/s
 
 </Tabs>
 
+### Snapshot configuration migration
+
+The latest snapshot YAML configurations introduced in dbt v1.9 and higher (such as [`snapshot_meta_column_names`](/reference/resource-configs/snapshot_meta_column_names) and [`dbt_valid_to_current`](/reference/resource-configs/dbt_valid_to_current)) are best suited for new snapshots. For existing snapshots, we recommend the following to avoid any inconsistencies to your snapshots:
```

Comment:
note i'm using 'latest' so it's more scalable for the long term. if i use 'new', it'll become outdated eventually i think.